lighthouse-core/test/audits/image-aspect-ratio-test.js

```diff
@@ -12,7 +12,7 @@ const assert = require('assert').strict;
 
 function generateImage(clientSize, naturalSize, props, src = 'https://google.com/logo.png') {
   const image = {src, mimeType: 'image/png'};
-  Object.assign(image, clientSize, naturalSize, props);
+  Object.assign(image, clientSize, naturalSize, {cssComputedObjectFit: 'fill', ...props});
```

Comment:
while we're here, can you modify this function to be an object literal (instead of using an assign).

```
return {
  src,
  mimeType: '...',
  ...clientSize,
  ...etc
};
```